### ISSUE TITLE
Is data augmentation allowed?

### ISSUE THREAD

**Issue by wjm-wjm** [eccv22-ood-workshop/ROBIN-dataset#5]
Hi,  is data augmentation allowed ? Can we generate some augmented data (like GAN) to train the model by using the proposed dataset?

**Comment by DTennant** [comment]
Hi,
We are preparing detailed rules for the competition, and will post it on Github within two weeks

**Comment by DTennant** [comment]
Hi,
Thanks again for your interest in our workshop, and we apologise for the late update.
I have update the README in this repo and update a link to the full test set and labels in the README, the codalab submission server will also be online within this week.

Here are some changes to the competition:

The Phase-1 of the competition will not be a code submission challenge, we have released all the test data and labels in this repo. And Phase-1 will last longer than original planed, we will ask each team to provide a description of their developing environment at the end of Phase-1, Phase-2 will still be code submission challenge.
We will be using Top-1 accuracy for image-classification, mAP@50 for object detection, and Acc@pi/6 for pose estimation as the metric, the IID test performance will also be considered as per request of the sponsor, we will penalize submissions that are significantly different in IID performance with our baseline.
The only limitation now is that the model should only be trained on the given training set and/or the ImageNet-1k dataset, no additional dataset is allowed. You can use any ensemble, data augmentation, or test-time training techniques.
Thanks again for your patience.

Best regards
